test/module/irohad/consensus/yac/supermajority_checker_test.cpp

```diff
@@ -38,48 +38,57 @@ TEST_F(SupermajorityCheckerTest, SuperMajorityCheckWithSize2) {
 }
 
 /**
- * @given 4 participants
+ * @given 6 participants
  * @when check range of voted participants
  * @then correct result
  */
 TEST_F(SupermajorityCheckerTest, SuperMajorityCheckWithSize4) {
-  log_->info("-----------| F(x, 4), x in {0..5} |-----------");
+  log_->info("-----------| F(x, 6), x in {0..7} |-----------");
 
-  int N = 4;
+  int N = 6;
   ASSERT_FALSE(checkSize(0, N));
```

Comment:
```
for (int i = 0; i < 100; ++i) {
  if (i == 5 or i == 6) {
    ASSERT_TRUE(checkSize(i, N));
  } else {
    ASSERT_FALSE(checkSize(i, N));
  }
}
```

Reply:
The following fixes might be reworked in similar way

Reply:
Please see https://github.com/hyperledger/iroha/blob/4baacb1e04e3eb92227aac0930e22a0a94869475/test/module/irohad/consensus/yac/supermajority_checker_test.cpp#L108-L121
I am going to PR this later

Reply:
ok then, but please don't insert such comments inside statements :(